src/app-layout/__tests__/runtime-drawers.test.tsx

```diff
@@ -72,6 +74,58 @@ describeEachAppLayout(({ theme, size }) => {
     expect(wrapper.findDrawersTriggers()).toHaveLength(2);
   });
 
+  test('update rendered drawers via runtime config', async () => {
+    awsuiPlugins.appLayout.registerDrawer({ ...drawerDefaults, resizable: true });
+    const { wrapper } = await renderComponent(<AppLayout />);
+    // the 2nd trigger is for tools
+    expect(wrapper.findDrawersTriggers()).toHaveLength(2);
+    const drawerTrigger = wrapper.findDrawerTriggerById(drawerDefaults.id);
+
+    if (theme !== 'classic' && size === 'desktop') {
+      const triggerParent = drawerTrigger?.getElement().parentElement;
```

Comment:
Use `!` instead of `?` in tests. When the value is unexpectedly empty, it will be easier to track down where the things started to go wrong.

Reply:
updated.